src/solvers/dgsem_tree/dg_2d_parabolic.jl

```diff
@@ -290,10 +291,11 @@ function prolong2boundaries!(cache_parabolic, flux_viscous,
     return nothing
 end
 
-function calc_viscous_fluxes!(flux_viscous, gradients, u_transformed,
+function calc_viscous_fluxes!(flux_viscous::Vector{Array{uEltype, 4}},
+                              gradients::Vector{Array{uEltype, 4}}, u_transformed,
```

Comment:
See above on `flux_viscous`, same comment for `gradients`

Reply:
At this place not, this is some left-over.